certs_governance/src/test.rs

```diff
@@ -203,6 +203,81 @@ fn test_initialize_contract_with_receivers_error() {
     )
 }
 
+#[test]
+fn test_distribute_with_distribution_limit_contract() {
+    let e: Env = Default::default();
+    let organization: Organization = Organization {
+        admin: Address::random(&e),
+        id_org: "12345".into_val(&e),
+    };
+    let distribution_limit = 6;
+    let cert_governance = create_cert_governance_contract_with_limit(
+        &e,
+        &distribution_limit,
+        &organization,
+        &true,
+        &Option::None,
+    );
+
+    pub const CID1: &str = "QmdtyfTYbVS3K9iYqBPjXxn4mbB7aBvEjYGzYWnzRcMrEC";
+    let address_receiver_1 = Address::random(&e);
+    let distribution_date: u64 = 1679918400;
+    cert_governance.distribute(
+        &organization.admin,
+        &address_receiver_1,
+        &"wallet_contract_id".into_val(&e),
+        &CID1.into_val(&e),
+        &distribution_date,
+    );
+    let receivers = cert_governance.receivers();
+    let cert_data = receivers.get(address_receiver_1).unwrap().unwrap();
+
+    assert_eq!(cert_data.status, Status::Distribute);
```

Comment:
Hmm maybe this variant of the **Status** Enumerable should be fixed, what do you think? 🤔 @Jenietoc 
```suggestion
    assert_eq!(cert_data.status, Status::Distributed);
```

Reply:
I think we can only use ten characters in this name

Reply:
Thanks for the clarification 🙏🏻